src/pages/using-spark/components/award.mdx

```diff
@@ -20,18 +19,14 @@ showcasing Awards with a disclaimer.
 
 ### Usage
 
-Use Award to showcase company accolades
-with a disclaimer. Award is a
-standard component of the
-[Footer](/using-spark/components/footer).
+Display company accolades and disclaimers on a page or in a [Footer](/using-spark/components/footer).
 
 ### Guidelines
 
-- The heading text is centered and can wrap to multiple lines.
-- Award images are stacked by default and
-expand to a side-by-side layout on larger viewports.
-- The Award images may be a clickable link.
-- Disclaimer content is hidden in a [Toggle](/using-spark/components/toggle) by default.
+- Heading text can wrap
+- Award images will stack by default and become side to side on larger viewports.
```

Comment:
"side to side" is the wrong usage. You move "side _to_ side", but two things next to each other are "side _by_ side"

Reply:
Good catch, thank you!